src/tests/ContentLoader.test.tsx

```diff
@@ -25,6 +25,10 @@ jest.mock('../lib/env.ts', () => ({
   }
 }))
 
+jest.mock('../lib/utils', () => ({
+  getVariant: jest.fn(x => '<p>Variant content</p>')
+}))
+
```

Comment:
Per [this comment](https://github.com/openstax/k12-apps-raise/pull/47#issuecomment-1272109242) we can remove this, and it's the reason for why the CI is currently failing. We get essentially the same coverage if you leave the existing tests but add the `variant: 'main'` to the content returned by the mock on line 11 above.